preferences/legacy/metadata/metadata.yaml

```
@@ -0,0 +1,14 @@
+---
+apiVersion: instancetype.kubevirt.io/v1beta1
+kind: VirtualMachinePreference
+metadata:
+  name: metadata
+  annotations:
+    tags: "hidden,kubevirt,legacy"
+    iconClass: "icon-linux"
```

```suggestion
    iconClass: "icon-other"
```
I suggest to use `icon-other` here.

---

All right. Done.